### DISCUSSION TITLE
How to use xdrv_17_rcswitch.ino driver in my IoT project?

### DISCUSSION THREAD

**Discussion by emanueloa** [arendst/Tasmota#18812] [General]
Hello guys!
I'm developing a project in which I receive, at an input, from an ESP-01, the signal from an RF433 transmitter. then it sent that same signal, through another ESP-01 port, to a controller.
I managed to implement it on an arduino uno, but I would like to use the ESP-01 with tasmota to use the web interface with 5 buttons, it also integrates with the home assistant and MQTT.
Could someone help me by explaining how to use the xdrv_17_rcswitch.ino diver and compile a cuatomized tasmota firmware?

**Comment by sfromis** [comment]
The RC-Switch driver is already including in the `tasmota-sensors` build, no custom build needed.
See https://tasmota.github.io/docs/RF-Protocol/
